### PR TITLE
Re-organize `remote` directory

### PR DESCRIPTION
**Pull request by shu-mutou**
Also, add descriptions for on-demand learners into READMEs.

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
For on-demand learners, add descriptions.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
[We](https://github.com/kubernetes-sigs/contributor-playground/blob/master/japan/OWNERS) can review and approve PRs for leaners. Added ourselves into OWNERS file under `remote` directory to help reviewing and approving.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by atoato88** [comment]
/lgtm
It's good for new contributor candidates. 😀

**Comment by atoato88** [comment]
Oh, sorry, but it's LGTM :smile:

File: remote/leaners/shu-mutou.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello kubenretes community!
```
Comment (oomichi) [inline remote/leaners/shu-mutou.md:1]: s/kubenretes/kubernetes/ ;-)

**Review by oomichi — COMMENTED** [review]
nice re-organization, but critical typo ;)

**Comment by oomichi** [comment]
Thanks for updating.

lgtm

**Comment by shu-mutou** [comment]
@nikhita @palnabarun Could you review this?

**Pull request closed without merging**